### ISSUE TITLE
[Bug]: Invisible Carpenters Table

### ISSUE THREAD

**Issue by Fyoncle** [CookieCNetwork/Chipped#309]
### Bug Description

Carpenters table just suddenly got invisible

### How to Reproduce?

1. No idea how it happened, it just got invisible, i think it happened after i go nether.
2. I play on Elysium Days modpack's version 3.5.0 btw

### Expected Behavior

Well, it should have been shown up like the Mason table or anything else.

### Version

3.0.6

### Mod Loader Version

1.20.1 - 0.15.11

### Mod Loader

Fabric

### Logs or additional context

![Screenshot_2](https://github.com/terrarium-earth/Chipped/assets/63975256/9dad1eac-c4d4-4421-89a7-23c528573657)

### Code of Conduct

- [X] I have searched the issue tracker and confirmed that the issue has not been reported
- [X] I have checked the FAQ (if one is present) and my issue is not listed
- [X] I have verified that I am using the latest version of the mod
- [X] I have verified that I have installed all the required dependencies for the mod
- [X] I have verified that I do not have Optifine installed. We will close issues if we detect Optifine is in your pack. We cannot fix Optifine issues. Please do not report issues with Optifine present.

**Comment by AlexNijjar** [comment]
is it only the mason table? do the other benches work? and is it consistent? i.e. if you restart your game does it reappear?

**Comment by Fyoncle** [comment]
> is it only the mason table? do the other benches work? and is it consistent? i.e. if you restart your game does it reappear?

Yeah only mason table disappeared, not sure when i restart i just re-placed it.
